pyproject.toml

```diff
@@ -10,9 +10,23 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "crick"
+description="High performance approximate and streaming algorithms"
+maintainers = [{name = "Jim Crist", email = "crist042@umn.edu"}]
```

Comment:
I believe Jim [no longer has time/desire to maintain crick](https://github.com/dask/crick/issues/25#issuecomment-1317701130). @fjetter I see dask/dask and distributed both have Matt as the maintainers, do we want to add/change this as well for crick?

Reply:
I suggest to keep this as is for now and fix the packaging without worrying about this too much. I'm fine changing this but I want Jim's ACK and frankly I don't know what else to put in here. We should probably setup an info@dask.org email address if that doesn't exist yet (@jacobtomlinson does something like that exist?)